content/articles/setting-up-kali-linux-for-dual-boot/index.md

```diff
@@ -0,0 +1,124 @@
+
+Select the pen drive listed on the menu to boot your PC from it. 
+
+Choose the `options` as guided below to avoid any errors during the installation process.
+
+- Select **Graphical Installation**.
+- Choose **Language**- in my case, `English`.
+- Select your **Region** and your preferred **Keyboard lab**.
+
+Wait for the installation set-up to load all components. This should take a short while.
+
+In the following setup, we'll use the recommended choices for an assured successful set-up.
+
+- Load missing firmware from removable media? - `No`.
+- Connect to your WiFi card if one is available - This is optional.
+
+Wait for the set-up to `configure all the Networks`. This process is also quick.
+
+Moving on, create a primary user for Kali Linux.
+
+- Set the `Host name` and `Domain name` as you wish - This is an optional step.
+- Enter your `username` and hit the continue button.
+- Enter and confirm the `password` entry.
+
+> **Note:** Remember your `username` and `password` since they are case sensitive and you will require them on every login session.
+
+### Creating disk partitions
+Since we have already created space for Kali Linux, we will do this manually. 
+- Select the `manual` option - This will display a list of all the drives on the PC.
+- Choose the drive with `free space` - This is the space we created. We will further create two partitions from the available free space.
+
+One of the partitions will be the `root partition` for Kali Linux and the other will be the `swap area`.
+
+First, let's create a **root partition** - This is where the operating system will be installed.
+
+Choose the free space option and select the options as shown below.
+
+- Select **create a new partition option**.
+- Allocate a space **95% or more** of the free space to the root partition.
+- Select the location of the new partition as the **beginning** of the drive.
+- Choose the File system as **Ext4 journaling**.
+- Mounting Point **/** - the root file system.
+
+**Note:** Click `done setting up the partition` before proceeding. Otherwise, you will have to start the partitioning process over again.
+
+### Creating a swap partition
+Swap area access virtual RAM if the system has less amount of physical memory. If your machine has 16GB of RAM or more, there is no need to create a swap partition.
+
+- Select the options same as creating a root partition.
+- Select the `swap area` option as the use of the partition.
+- Remember to select `done setting up the partition`.
+
+At this point, you have successfully created a partition.
+Click `Finish partitioning and write changes to disk` then hit the continue button.
+
+On the **write changes to disk** option, select `Yes` and wait for the installer set-up to `install the base system`.
+
+### Software selection
+Here you have to choose your favorite desktop environment. In my case, I'll' choose the `Xfce (Kali default desktop environment)`.
```

Comment:
Remember the reader just installed Kali Linux. They don't have a favorite desktop environment yet. Consider mentioning a few (if not all) environments and their differences.
Try to answer the question - Why this and not the other one?